make/modules/java.base/lib/CoreLibraries.gmk

```diff
@@ -53,19 +53,22 @@ endif
 $(eval $(call SetupJdkLibrary, BUILD_LIBJAVA, \
     NAME := java, \
     OPTIMIZATION := HIGH, \
+    EXTRA_HEADER_DIRS := \
+        libnio/ch, \
```

Comment:
This will introduce a source code dependency from libjava to libnio. It might be the correct thing to do, but I just want to highlight this. 

Maybe the libnio/ch code also should move to libjava? Or, perhaps at least the header files. Conceptually, I'd feel better about having a dependency on libnio as a "downstream" library on libjava source code, than the other way around.

Reply:
> This will introduce a source code dependency from libjava to libnio. It might be the correct thing to do, but I just want to highlight this.

I'll look into improving it.

Reply:
Fixed in 7e8a02e